### ISSUE TITLE
Localized weather data

### ISSUE THREAD

**Issue by pylerSM** [chubin/wttr.in#10]
Is it possible? Perharbs as auto detection/ "-locale_code" argument?

If weather server does not provide it, we can translate it manually I think.

**Comment by chubin** [comment]
Yes, I have such plans, and I will ask for help in twiiter (but not yet)

**Comment by rcspam** [comment]
I use a workaround to display weather in my language (french): I pipe curl output in a sed file:
$ curl http://wttr.in  2>/dev/null | sed -f wttr_sed_file
You can adapt it for your language easily. You completed it as work progresses.
[Here](https://dl.dropboxusercontent.com/u/2677320/wttr-sed) is the link to download it

**Comment by schachmat** [comment]
Heyho @chubin,

I just noticed your wttr.in instance already has localization support, but I can not find the code in the repository. Could you add it? People can better help with the translations then.
